### ISSUE TITLE
Feedback: 27/08 Playtest

### ISSUE THREAD

**Issue by trundler-dev** [apetavern/sbox-fortwars#53]
### needs fix
- [ ] Repair tool can be used to kill own team
- [ ] Announcer is annoying (remove?)
- [ ] Missing material for rocket launcher explosion
- [ ] Spawn forcefields have a weird delay when passing through
- [ ] Ammo and health kits do not work
- [ ] Give ammo and health kits team indicators
- [ ] Glow shader is now gone, needs replaced
- [ ] Switch voice prox to global, better voice UI
- [ ] Walking sound
- [ ] Music
- [ ] Team auto balancing
- [ ] Fix chat
- [ ] Nerf the rpg
- [ ] Weird movement sometimes when walking on spawned blocks
- [ ] Muzzle flash too big - it's impossible to see who you're spraying down while firing

### feature ideas
- [ ] Rework ammo and health kits to be more like BF2042
- [ ] Jump pads that only work for your own team (strategic placement to allow your team to jump over your own obstacles)

**Comment by xezno** [comment]
Rework ammo and health kits to be more like BF2042 done in f78ae45a02cf5db39304381c84d0ab429bc52fcb

**Comment by xezno** [comment]
Changed a bunch of RPG stuff in 36f5c58986851a6b8bfe591f86dc67d02148ce7b. I've deliberately targeted all of the things that make it useful as an infantry weapon (high damage output vs players, lots of ammo, quick reload time, quick fire rate) and tweaked the weapon around that in order to try and make it less viable against players and more viable against buildings.
Something tells me this will probably need another pass or two later on, though. :D

**Comment by xezno** [comment]
Chat is probably fixed as of 5586bfd538020c9ba2eb3e4fdcc51f4657b4c13a but needs testing so I'm not ticking it off yet

**Comment by ShadowBrian** [comment]
Chat seems to work. The backdrop is still black though which looks kinda shit, I recall it being a blur before?

**Comment by xezno** [comment]
Yeah it was, I changed it because I felt like the chat was a bit too difficult to read before

**Comment by ShadowBrian** [comment]
Could just make it a light grey instead or something maybe

**Comment by xezno** [comment]
Walking sounds fixed in 858f2445106588fdd2307d3275766da0b7d25891.. turns out they were there, just so quiet you couldn't hear them. Also made it so they didn't apply when sliding

**Comment by xezno** [comment]
Repair tool fixerino in d7743513a003e35efc54e26063d821bbfe777c0e

**Comment by xezno** [comment]
Glows fixed in 468335c71c241a1280ebf886f1a79ac6a895e653

**Comment by xezno** [comment]
Team balancer added in 715a8f97fca83b72d07ad861f12a8bbda20f681f, this time it'll only balance the players it needs to rather than shuffling everyone around

**Comment by xezno** [comment]
Tweaked announcer in 65f72fd777b2794618079b33dd279dc3cda74db9, he shouldn't talk over himself anymore, should be less annoying.
